### PR TITLE
K8SPXC-219 Helm 3 support

### PR DESCRIPTION
**Pull request by cap1984**

### CODE REVIEW

File: charts/pxc-operator/Chart.yaml [hunk 1]
```diff
@@ -1,9 +1,9 @@
 apiVersion: v1
 appVersion: "1.3.0"
-description: A Helm chart for Deploying the Percona XtraDB Cluster Operator Kubernetes
+description: A Helm 3 chart for Deploying the Percona XtraDB Cluster Operator Kubernetes
```
Comment (paulczar) [inline charts/pxc-operator/Chart.yaml]: I don't think we need to specify `Helm 3` in the description.  This chart should continue to work for `Helm 2` as well, with the additional instructions of `kubectl apply -f crd/` beforehand.

**Comment by paulczar** [comment]
You shouldn't need to copy the `crd` to `charts/pxc-db/crds/crd.yaml` ... only the operator chart itself should have the crd.

**Comment by cap1984** [comment]
> You shouldn't need to copy the `crd` to `charts/pxc-db/crds/crd.yaml` ... only the operator chart itself should have the crd.

Unfortunately in case of missing operator, Kubernetes will throw an error regarding PXC object version types

File: charts/pxc-operator/Chart.yaml (hunk 1 above)
Comment (cap1984) [reply charts/pxc-operator/Chart.yaml]: ok

**Comment by Jancis** [comment]
We're using namespace limited SA for deployment and since `pxc-db` also wants to install crd, our deployment fails. `pxc-db` crd is identical to `pxc-operator` crd, it's already set up, please don't have it in `pxc-db` (having an operator ir a pre-requisite).

**Pull request merged**